backend/init_db.py

```
@@ -124,6 +124,20 @@ def migrate_db():
             except Exception:
                 pass
 
+            # Add parent_issue_id column for deduplication
+            try:
+                conn.execute(text("ALTER TABLE issues ADD COLUMN parent_issue_id INTEGER"))
+                print("Migrated database: Added parent_issue_id column.")
+            except Exception:
+                pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)